deployment/release.sh

```diff
@@ -0,0 +1,9 @@
+#  version="USER INPUT"
+read -p "Enter version number: " version
+
+git checkout master
+git flow release start $version
+git flow release publish $version
+git flow release finish $version
+git push origin --tags
+git push
```

Comment:
why do you need to push twice?

Reply:
`git push` doesn't push the tags so I have to do an explicit tag push

Reply:
yeah, but what use is the second `git push`?

Reply:
Oh nice I thought it just pushed the tags!! I'll remove the extra push